### PR TITLE
Subtle messages balloon alert

### PR DESCRIPTION
**Pull request by Therealdoooc213**
## About The Pull Request
Subtle messages (AND HEADSET MESSAGES) appear with a balloon alert saying "You hear a voice in your head.." now

## Why It's Good For The Game
Subtle messages are very hard to miss, people aren't always looking at the bottom right of their screen, the balloon alert will get people to look at the bottom of their screen then see the real message.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

https://github.com/user-attachments/assets/88a47427-a51c-4904-b372-09af9d4f57df

https://github.com/user-attachments/assets/e2b45bf2-7279-4341-bd44-70faa4051f64

</details>

## Changelog
:cl:
tweak: Subtle messages, telepathy, and headset messages now show a balloon alert
/:cl:

### CODE REVIEW

**Comment by Marshmellow105** [comment]
great change, could you add this functionality to headset messages? you send them through the player panel.

**Review by EvilDragonfiend — APPROVED** [review]

**Review by Rukofamicom — APPROVED** [review]

**Review by PowerfulBacon — COMMENTED** [review]
This also needs to apply to telekinetic messages, and why not show the message? If you are showing it in balloon chat people are going to be confused when they see "you see a voice in your head" and there's no actual message other than hidden away in the chat where you are diverting their attention away from

**Comment by Therealdoooc213** [comment]
> This also needs to apply to telekinetic messages, and why not show the message? If you are showing it in balloon chat people are going to be confused when they see "you see a voice in your head" and there's no actual message other than hidden away in the chat where you are diverting their attention away from

Its supposed to be a indicator to divert your attention to the chat. Most people will get the hint of the balloon alert and go to check the chat box imo. I thought telekinesis already did that but i'll add it now.

**Comment by PowerfulBacon** [comment]
The only reason is to actually make people look at them anyway. You wouldn't need the ratvar span if the message appeared directly to the user

**Comment by Therealdoooc213** [comment]
Also, you cant look back on balloon alerts. Maybe an admin sent you the nuke codes for example but you dont remember them because it appeared in a balloon alert.

**Comment by TsunamiAnt** [comment]
Good point actually. If we want custom themed messages we can always use the narrates anyways.

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -38,6 +38,7 @@
 		if (usr.client)
 			if(usr.client.holder)
 				to_chat(M, "<i>You hear a voice in your head... <b>[msg]</i></b>")
+				M.balloon_alert(M, "you hear a voice in your head...")
```
Comment (Archanial) [inline code/modules/admin/verbs/randomverbs.dm]: Why is that sentence starting with lower case?

**Review by Archanial — DISMISSED** [review]
Why not display full message as a baloon instead? It should wrap anyway

**Comment by Therealdoooc213** [comment]
> Why not display full message as a baloon instead? It should wrap anyway

Look back on the previous conversation

**Comment by Archanial** [comment]
Just display the message both as baloon and in chat. What's the issue with that?
The idea of chatmessages is that you don't look at the chat.

**Comment by EvilDragonfiend** [comment]
> Just display the message both as baloon and in chat. What's the issue with that? The idea of chatmessages is that you don't look at the chat.

You shouldn't know where that message comes from.

**Comment by Archanial** [comment]
> You shouldn't know where that message comes from.
So? You don't know where this message comes from because it is sent by admins. If anything you could add like "..." at the start and do italics to differentiate this message from the rest.

**Comment by Tsar-Salat** [comment]
> Subtle messages are very hard to miss

Are very *easy* to miss you mean? Whar?

**Comment by Tsar-Salat** [comment]
> Why not display full message as a baloon instead?

Because displaying a shitton of text in the balloon looks terrible. 

Balloons are useful as a visual indicator, however one needs to be wary of putting way more text than necessary,

As it stands, this PR is just to remind the player they were sent a message so they check the chatbox. There really is no reason to clutter the screen with it. (esp if the player is mid conversation when we admins message them)

**Review by Tsar-Salat — APPROVED** [review]
I see two approvals, I will approve as well. 

This is very useful personally as admin for having newer players actually look when I message them.

**Pull request merged**